src/components/DayInformation/DayInformation.tsx

```diff
@@ -107,36 +111,89 @@ function DayInformation({ day }: Props) {
-                                    }}
-                                ></FormCheck>
-                            </CheckboxItem>
-                            <Button
-                                variant="danger"
-                                className="ms-2"
-                                onClick={() => {
-                                    TaskController.removeTaskFromDate(day, task.id);
-                                    let dayTasks: Task[] = TaskController.getTasksForDate(day);
-                                    setDayTasks(dayTasks);
-                                }}
-                            >
-                                <FontAwesomeIcon icon={solid('trash')} />
-                            </Button>
+
+                            {isEditing === task.id && (
+                                <Form className="d-flex flex-grow-1">
+                                    <Form.Control
+                                        type="text"
+                                        name="description"
+                                        maxLength={196}
+                                        className="me-2"
+                                        defaultValue={task.description}
+                                        onChange={(e) => {
+                                            setEditedDescription(e.target.value);
+                                        }}
+                                    />
+                                    <Button
+                                        type="submit"
+                                        variant="success"
+                                        className="me-2"
+                                        onClick={() => {
+                                            TaskController.editTask(day, task.id, editeddescription)
+                                            let dayTasks: Task[] = TaskController.getTasksForDate(day);
+                                            setDayTasks(dayTasks);
+                                        }}
+                                    >
+                                        <FontAwesomeIcon icon={solid('check')} />
+                                    </Button>
+                                    <Button
+                                        variant="danger"
+                                        onClick={() => {
+                                            setIsEditing('');
+                                        }}
+                                    >
+                                        <FontAwesomeIcon icon={solid('xmark')} />
+                                    </Button>
+                                </Form>
+                            )}
+
+                            {isEditing !== task.id && (
+                                <><CheckboxItem style={(["bae", "bae's", "baby"].some(x => task.description.includes(x)))  ? {backgroundColor:'#F8C8DC'} : {}}>
```

Comment:
add the same class as above instead of a style

Reply:
actioned.